code/modules/customitems/item_defines.dm

```diff
@@ -2068,7 +2069,7 @@ obj/item/clothing/suit/storage/hooded/fluff/make_poncho //Raincoat Poncho - M.A.
 
 /obj/item/weapon/deck/tarot/fluff/klavdiya_cards //Adhomian Divination Cards Deck - Klavdiya Tikhomirov - alberyk
 	name = "adhomian divination cards deck"
-	desc = "An adhomian deck of divination cards, used to read the fortunte or play games."
+	desc = "An adhomian deck of divination cards, used to read the fortune or play games."

```

Comment:
`the fortune` is grammatically incorrect. Better phrasing might be: `An adhomian deck of divination cards, used to read one's fortune or play games.`